src/app/components/modal/favorite-modal/favorite-modal.component.scss

```diff
@@ -0,0 +1,31 @@
+@import "../../../../styles/variables.scss";
+
+.c-modal {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    &__description{
+        &--title{
+            color: $color-gray-700;
+            font-size: $font-h5;
+            font-weight: $font-bold;
+        }
+        &--text{
+            color: $color-gray-300;
+            font-size: $font-body-small;
+            font-weight: $font-semibold;
+        }
+    }
+    &__buttons{
+        display: flex;
+        width: 60%;
+        flex-direction: column;
+        gap: 10px;
+        button{
+                font-size: $font-button;
+                font-weight: $font-semibold;
+        }
```

Comment:
Keep a consistent indentation in SCSS files. You can use the Prettier Visual Code extension that will format the code on save.